UI/runScan.php

```diff
@@ -0,0 +1,100 @@
+<!-- 
+    When index.php is compiled and run on the terminal, it will launch this runScan.php script with messages visible.
+    Note: output messages built-into the crawler script are not visible on the server-side while running localhost:<port#> 
+    (i.e. during demonstration). However, at the end of the scan, output will be logged to the UI console available to view 
+    using: Fn F12.
+ -->
+<?php
+
+    // Allow capturing output buffer
+    ob_implicit_flush(true);
+    
+    // Locate and parse configuration file
+    // $config = parse_ini_file("./config.ini");
+    // $username = $config["username"];
```

Comment:
The username for our app is different from the username for the database. The database credentials (user/pass) are fixed; any sort of dynamic validation/setting is beyond our scope. The username that we are passing around through code and that the user inputs on the login screen is solely their user/pass for our app, which tells the DB what client they are associated with.